src/components/Campaign.tsx

```diff
@@ -121,7 +122,14 @@ export const Campaign = () => {
             </EditorTopBar>
           </Header>
           <Footer>
-            <EditorBottomBar {...continuationUrls}>
+            {campaignContentMutation.isLoading ? (
+              <SavingMessage>
+                <FormattedMessage id="saving" />
+              </SavingMessage>
```

Comment:
If we always use the same message for saving, what do you think about moving the formatted message inside `SavingMessage` component?

```suggestion
              <SavingMessage />
```

Reply:
Solved